### ISSUE TITLE
Lua error in 5.0.0

### ISSUE THREAD

**Issue by filliph** [derfloh205/LoadoutReminder#6]
```
1x ...eLoadoutReminder/Modules/Options/Options.lua:576: attempt to concatenate local 'difficulty' (a nil value)
[string "@LoadoutReminder/Modules/Options/Options.lua"]:576: in function `GetPerBossOptionKey'
[string "@LoadoutReminder/Modules/Options/Options.lua"]:585: in function `HasRaidLoadoutsPerBoss'
[string "@LoadoutReminder/Main/LoadoutReminder.lua"]:95: in function `CheckInstanceTypes'
[string "@LoadoutReminder/Main/LoadoutReminder.lua"]:68: in function `CheckSituations'
[string "@LoadoutReminder/Main/LoadoutReminder.lua"]:244: in function `?'
[string "@LoadoutReminder/Main/LoadoutReminder.lua"]:5: in function <...faceLoadoutReminder/Main/LoadoutReminder.lua:5>
```

**Comment by filliph** [comment]
Happens when entering the raid and prevents the addon from functioning.

**Comment by derfloh205** [comment]
should be fixed in 5.0.1
